pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -179,7 +179,10 @@ func Execute(configFile io.Reader) {
 func NewServer(ctx context.Context, dockerConfig *configuration.Configuration, extraConfig *registryconfig.Configuration) (*http.Server, error) {
 	setDefaultLogParameters(dockerConfig)
 
-	registryClient := client.NewRegistryClient(clientcmd.NewConfig().BindToFile(extraConfig.KubeConfig))
+	clientConfig := clientcmd.NewConfig()
+	clientConfig.CommonConfig.AcceptContentTypes = "application/vnd.kubernetes.protobuf,application/json"
+	clientConfig.CommonConfig.ContentType = "application/vnd.kubernetes.protobuf"
+	registryClient := client.NewRegistryClient(clientConfig.BindToFile(extraConfig.KubeConfig))
```

Comment:
BindToFile ignore whatever the original client config is, it looks like.

Reply:
It doesn't ignore it, but this is not working the way I would expect.  Still seeing json to the server in the metrics capture.

Reply:
@smarterclayton hm. i just refactored it to add the content setting after the bindtofile call and pushed, but it sounds like you don't think that'll matter.